project/BuildSettings.scala

```diff
@@ -5,7 +5,7 @@ import sbtassembly.AssemblyPlugin.autoImport._
 object BuildSettings {
   val buildOrganization = "cfpb"
   val buildVersion      = "2.0.0"
-  val buildScalaVersion = "2.13.8"
+  val buildScalaVersion = "2.12.15"
```

Comment:
@lchen-2101  is this version rollback intentional?

Reply:
yes sir, 2.12.x to 2.13.x is a major version update, there are other dependencies that no longer works and needs to be updated alongside the 2.13.x update; there are also other code breaking changes by going to 2.13.x; I couldn't build successfully with the change of just the scala version update.